GoogleChromeLabs/wp-sitemaps
# Bail early if search engines are discouraged

### Issue Number
Fixes #111.
Fixes #123.

### Description

Does not do anything for private sites.

### Type of change
Please select the relevant options:


### Test Instructions

1. Activate plugin
1. Make site private
1. Verify that sitemap is not being generated

### Acceptance criteria
